### ISSUE TITLE
Use rimraf or shelljs

### ISSUE THREAD

**Issue by ghost** [watilde/npmbrew#18]
What about using [`rimraf`](https://github.com/isaacs/rimraf) instead of rmrf, or even better [shelljs](https://github.com/arturadib/shelljs).

Another option:

``` js
require('child_process').exec('rm -Rf dir', function(err, out) { 
    // ...
}); 
```

But would that work on Windows?

**Comment by watilde** [comment]
Yeah I agree to use [rimraf](https://github.com/isaacs/rimraf) just because npm used that. So I'll fix this issue. Thanks!

**Comment by ghost** [comment]
I have the PR almost ready, let me work on this one plus a few other things you can see if you wait a bit.

**Comment by watilde** [comment]
OK!  I'm looking forward to it :smiley_cat:

**Comment by ghost** [comment]
Cool, I also want to add commander.js instead of the current manual approach. What do you think?

**Comment by ghost** [comment]
Closed #21

**Comment by watilde** [comment]
@bucaran It's really useful if npmbrew needs complex options, so I thought it isn't necessary for now.
